test/handshake_helper.c

```diff
@@ -1038,6 +1039,19 @@ static HANDSHAKE_RESULT *do_handshake_internal(
     if (session_out != NULL)
         *session_out = SSL_get1_session(client.ssl);
 
+    if (SSL_get_server_tmp_key(client.ssl, &tmp_key)) {
+        int nid;
```

Comment:
Some people seem to want a blank line here, but our style document doesn't seem to say anything about it.